.pubnub.yml

```diff
@@ -87,6 +87,22 @@ features:
     - PUBLISH-ASYNC
     - PUBLISH-REPLICATION-FLAG
     - PUBLISH-MESSAGE-TTL
+  subscribe:
+    - SUBSCRIBE-CHANNELS
+    - SUBSCRIBE-CHANNEL-GROUPS
+    - SUBSCRIBE-PRESENCE-CHANNELS
+    - SUBSCRIBE-PRESENCE-CHANNELS-GROUPS
+    - SUBSCRIBE-WITH-TIMETOKEN
+    - SUBSCRIBE-FILTER-EXPRESSION
+    - SUBSCRIBE-PUBLISHER-UUID
+    - SUBSCRIBE-PUBSUB-V2
+    - SUBSCRIBE-MESSAGE-ACTIONS-LISTENER
+  presence:
+    - PRESENCE-HERE-NOW
+    - PRESENCE-WHERE-NOW
+    - PRESENCE-SET-STATE
+    - PRESENCE-GET-STATE
+    - PRESENCE-HEARTBEAT
```

Comment:
@Xavrax it is confusing because section, but I think we need this one as well: `PRESENCE-DELTAS` - our subscribe parser able to handle interval update of present users. 
@techwritermat is this flag about interval update of presence (when we receive set of joined/left and timeout)?

Reply:
DM'd you